Assets/VRTK/Scripts/VRTK_SimplePointer.cs

```diff
@@ -19,10 +19,13 @@ namespace VRTK
     public class VRTK_SimplePointer : VRTK_WorldPointer
     {
         public float pointerThickness = 0.002f;
+        [Tooltip("Maximum distance the pointer can reach")]
         public float pointerLength = 100f;
         public bool showPointerTip = true;
         public GameObject customPointerCursor;
         public LayerMask layersToIgnore = Physics.IgnoreRaycastLayer;
+        [Tooltip("Scales the pointer proportional to this (usually CameraRig)")]
+        public GameObject scaledReference;
```

Comment:
Being flexible is nice, but I wonder if it would be better to auto set this to the camera at startup if the player hasn't filled in an override.